tools/testowners/main.go

```diff
@@ -35,20 +35,15 @@ type TestEvent struct {
 
 var (
 	CodeOwners []string
+	Format     string
 )
 
 func init() {
 	flag.StringSliceVar(&CodeOwners, "code-owners", []string{}, "Use the code owners defined in these files for --log-code-owners")
+	flag.StringVar(&Format, "format", "testfailures", "The format of the input. One of 'testfailures' (default) or 'plain'")
```

Comment:
nit: I'd replace `testfailures` with `gotest` as this is standard `go test` output. This is a bit clearer than "test failures" which might imply some other format.


```suggestion
	flag.StringVar(&Format, "format", "gotest", "The format of the input. One of 'testfailures' (default) or 'plain'")
```

Reply:
agreed, thanks